tests/python/frontend/tflite/test_forward.py

```diff
@@ -14,20 +14,30 @@
 # KIND, either express or implied.  See the License for the
 # specific language governing permissions and limitations
 # under the License.
-# pylint: disable=import-self, invalid-name, unused-argument
+# pylint: disable=unused-argument, disable=import-outside-toplevel, inconsistent-return-statements
```

Comment:
1. import-outside-toplevel can be remove when tflite https://github.com/apache/tvm/blob/9a4d80c5fe7d756baa83708fd9679c7ae0fac195/tests/python/frontend/tflite/test_forward.py#L190-L199
2. inconsistent-return-statements https://github.com/apache/tvm/blob/9a4d80c5fe7d756baa83708fd9679c7ae0fac195/tests/python/frontend/tflite/test_forward.py#L1976-L1995